Automattic/wp-calypso
# Render a list of developer features

Resolves https://github.com/Automattic/dotcom-forge/issues/5233

## Proposed Changes
We are moving "I am developer" checkbox from "Profile" to "Developer features" page - https://github.com/Automattic/wp-calypso/pull/86558.
Additionally we need to show the list of features which are available for developers, to provide the info of the checkbox "I am developer".
With this PR we are adding the list of that features to the page "Developer features".

## Testing Instructions
0) Checkout to the branch locally or open live branch
1) Open `http://calypso.localhost:3000/me/developer`
2) Assert that you see features and styles looks according to the design huXuof0X5yU9bqYvy6BPBW-fi-1970_1870
3) Assert that mobile version also looks goo according to the design huXuof0X5yU9bqYvy6BPBW-fi-1970_1870
4) Assert that loading-placeholder for "I am dev" checkbox looks also good: <br />![Screenshot 2024-01-23 at 12 43 09](https://github.com/Automattic/wp-calypso/assets/5598437/0625abb3-d9e4-478f-bd68-988c5e593d2f)


## The UI result (primarily for translators-team, to understand what it looks like)
![Screenshot 2024-01-23 at 12 48 55](https://github.com/Automattic/wp-calypso/assets/5598437/7e438699-d340-4e0d-bee6-8a550c45e8f0)
